coreservices/keycloak/src/CatenaX.NetworkServices.Keycloak.Factory/KeycloakFactory.cs

```diff
@@ -0,0 +1,20 @@
+using Keycloak.Net;
+using Microsoft.Extensions.Options;
+
+namespace CatenaX.NetworkServices.Keycloak.Factory
+{
+    public class KeycloakFactory : IKeycloakFactory
+    {
+        readonly KeycloakSettingsMap _Settings;
+        public KeycloakFactory(IOptions<KeycloakSettingsMap> settings)
+        {
+            _Settings = settings.Value;
+        }
+
+        public KeycloakClient CreateKeycloakClient(string instance)
+        {
+            KeycloakSettings settings = _Settings[instance];
+            return new KeycloakClient(settings.ConnectionString, settings.User, settings.Password, settings.AuthRealm);
```

Comment:
Do we now have a single auth realm?
if yes, we can also just use the nuget package. 
We only copied the keycloak branch because we the master auth realm to change things in other realms